src/components/HeaderDashboard/HeaderDashboard.tsx

```diff
@@ -1,36 +1,28 @@
 import React from 'react';
-import { Nav, Navbar } from 'react-bootstrap';
-import { useRecoilState } from 'recoil';
-import DarkModeToggle from 'react-dark-mode-toggle';
+import { Navbar } from 'react-bootstrap';
+import { useRecoilValue } from 'recoil';
 import { useNavigate } from 'react-router-dom';
 
 import { darkModeState } from '@store';
 
+import logo from '@assets/images/logo.svg';
 import ZinZenTextLight from '@assets/images/LogoTextLight.svg';
 import ZinZenTextDark from '@assets/images/LogoTextDark.svg';
-import Logo from '@assets/images/zinzenlogo.png';
+import ArrowIcon from '@assets/images/ArrowIcon.svg';
 import '@translations/i18n';
 import './HeaderDashboard.scss';
 
 export const HeaderDashboard = () => {
   const navigate = useNavigate();
```

Comment:
@GithinjiHans i think the TCS are failing because of the navigate function that you are using here since this component is called in the feedback page. I think we need to mock it